### PR TITLE
W-14778235 add hc 2.22 release folder and index updates

### PR DESCRIPTION
**Pull request by tasmoore**
**What Needs Review**
Only the following files have new content and need review:
- index.adoc: 1) QHIN use case name and description 2) The Exchange link to the use case
- antora.yml: Ensure 2.22 is the correct release version

**PR Overview**
This PR covers the following:
- Added Healthcare 2.22 folder
- Added QHIN use case and short description to index.adoc file
- Updated antora file to 2.22

Note: Release notes are being drafted in a different repo and shared via a different PR.

### CODE REVIEW

File: healthcare/2.22/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%207%20-%20Population%20health%20management/[Population health management^] -  Convert and surface valuable population health data into Amazon HealthLake to improve clinical outcomes.
+* https://www.anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%207%20-%20Qualified%20health%20information%20network%20integration/[Qualified Health Information Network Integration] - Streamline patient onboarding and remove manual data entry by integrating with external organizations to gather and ingest patient record data.
```
Comment (ppalu) [inline healthcare/2.22/modules/ROOT/pages/index.adoc]: can we say "ingest patient data" towards the end of the sentence?
@rstastny9 - any thoughts?

Comment (rstastny9) [inline healthcare/2.22/modules/ROOT/pages/index.adoc]: Can we make this line read: 
"Streamline patient onboarding and remove manual data entry by integrating with external healthcare organizations to gather and ingest patient data."

**Review by rstastny9 — CHANGES_REQUESTED** [review]

File: healthcare/2.22/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (rstastny9) [reply healthcare/2.22/modules/ROOT/pages/index.adoc]: "Streamline patient onboarding and remove manual data entry by integrating with external healthcare organizations to gather and ingest patient data." 

This was my review ^^

Comment (ppalu) [reply healthcare/2.22/modules/ROOT/pages/index.adoc]: @tasmoore - use case page number needs to be 8 instead of 7

Comment (ppalu) [reply healthcare/2.22/modules/ROOT/pages/index.adoc]: @rstastny9 - Is "Qualified health information networks integration" use case page name?

Comment (rstastny9) [reply healthcare/2.22/modules/ROOT/pages/index.adoc]: I think it should be "Qualified Health Information Networks integration" - all caps

**Review by luanamulesoft — APPROVED** [review]

**Pull request merged**